app/models/ontology.rb

```diff
@@ -147,6 +153,17 @@ def title
     name? ? iri : nil
   end
 
+  def self.find_with_locid(locid, iri=nil)
```

Comment:
Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.